### PR TITLE
Modified deployment instructions to use shared fabrikam-worklaod repo

### PR DESCRIPTION
**Pull request by magrande**

### CODE REVIEW

**Review by ferantivero — CHANGES_REQUESTED** [review]
promptly blocking this one til we are done with the advanced one.

File: deployment.md [hunk 1]
```diff
@@ -33,8 +33,8 @@ Set environment variables.
 
 ```bash
 export SSH_PUBLIC_KEY_FILE=[YOUR_RECENTLY_GENERATED_SSH_RSA_PUBLIC_KEY_FILE_HERE]
-export LOCATION=[YOUR_LOCATION_HERE]
-export RESOURCE_GROUP=[YOUR_RESOURCE_GROUP_HERE]
+export LOCATION=eastus2
+export RESOURCE_GROUP=rg-shipping-microservices
```
Comment (v-fearam) [inline deployment.md:37]: The constants set are ok or they are here by mistake?

**Review by v-fearam — APPROVED** [review]

**Review by ckittel — APPROVED** [review]
Looks pretty well aligned to the other, but I'll defer to @ferantivero for anything else that should be addressed.

**Review by ferantivero — CHANGES_REQUESTED** [review]

File: azuredeploy-prereqs.json [hunk 2]
```diff
@@ -1,161 +0,0 @@
-{
```
[the remark this answers is not in the document]
Comment (v-fearam) [reply azuredeploy-prereqs.json:1]: about CI/CD, we agree to delete that from Basic. Neil confirms that he has deleted the references (Chad asked Neil about that last week), so next step is deleting from the repo.

[the remark this answers is not in the document]
Comment (ferantivero) [reply azuredeploy-prereqs.json:1]: @magrande please let's get rid of the following section from the `deployment.md`:

### Optional: Set up automated CI/CD for dev, test, qa and production with Azure DevOps

Additionally delete `yml` + `json` related to CI/CD from https://github.com/mspnp/fabrikam-dronedelivery-workload (it doesn't need to take precedence).

**Review by ferantivero — APPROVED** [review]

**Pull request merged**